doc/source/changed_in_8.0.rst

```diff
@@ -0,0 +1,171 @@
+.. _changed_in_8.0:
```

Comment:
If this file is per-per branch, having a version number in the file name is redundant

Reply:
This is the format which was used in previous releases, such as *changed_in_5.7* and so on. Nevertheless, just removing the release number makes the file name too general. I would suggest to rename it to *changed_in_release.rst*